tests/apple-exporter/components/test-class-image.php

```diff
@@ -278,6 +278,38 @@ public function testRelativeSrc() {
 		$this->assertEquals( 'anchored-image', $result['layout'] );
 	}
 
+	/**
+	 * Tests dark color setting for image captions
+	 *
+	 * @access public
+	 */
+	public function testDarkColors() {
+		$this->set_theme_settings(
+			[
+				'caption_color_dark'       => '#abcdef',
+			]
+		);
+
+		$html = <<<HTML
+<figure>
+	<img src="http://someurl.com/filename.jpg" alt="Example">
```

Comment:
I've been switching to using `example.org` everywhere because it's reserved by IANA for this purpose.
```suggestion
	<img src="https://example.org/filename.jpg" alt="Example">
```